app/models/user_seen_subject.rb

```diff
@@ -23,6 +22,11 @@ def self.count_user_activity(user_id, workflow_ids=[])
     scope.sum("cardinality(subject_ids)")
```

Comment:
could we just summate the per workflow call here now instead of running 2 different db calls?